.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,25 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/python
+{
+	"name": "Python 3",
+	// Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
+	"image": "mcr.microsoft.com/devcontainers/python:0-3.11",
+
+	// Features to add to the dev container. More info: https://containers.dev/features.
+	// "features": {},
+	"features": {
+    "azure-cli": "latest"
```

Comment:
I believe you will need to docker-in-docker devcontainer feature too.
If you try and do `make serve-local` right now, it fails inside a dev container because docker isn't present.